packages/typography/src/Link.tsx

```diff
@@ -59,9 +59,10 @@ const arrowRightIconHover = css`
 `;
 
 const openInNewTabStyles = css`
-  margin-bottom: 4px;
-  margin-left: -1px;
-  margin-right: -2px;
+  position: relative;
```

Comment:
If these styles needed to change, are there other styles that we should be cognizant of updating?

Reply:
Nope – this is specifically because previously this specific icon didn't take up the full 16x16 artboard, and now it does. This is the only place we use this glyph. It's possible we should change to a major version for the icon updates, however 🤔

Reply:
Ah got it